test/linear_combination.jl

```diff
@@ -168,14 +168,9 @@
         @testset "all cols" begin
             nt = (a = [1, 2, 3], b = [4, 5, 6])
             lc = LinearCombination([1, -1])
-            @test FeatureTransforms.apply(nt, lc) == [-3, -3, -3]
-            @test lc(nt) == [-3, -3, -3]
-        end
-
-        @testset "dims not supported" begin
```

Comment:
Curious why this was removed

Reply:
`apply(data, ::LinearCombination, ...)` previously didn't have a `dims` kwarg so this returned `MethodError`.

But now since it has to be able to accept arbitrary `kwargs` (to accept `header`) this no longer occurs.

I couldn't set default `header` in the usually `apply` because it isn't know until the transform is applied how many cols the result will have. But in this case we do know (it'll be 1) so I could set `header` to be explicit.